projects/admin/src/app/record/detail-view/document-detail-view/entities-related/entities-related.component.html

```diff
@@ -0,0 +1,31 @@
+<!--
+  RERO ILS UI
+  Copyright (C) 2019-2023 RERO
+
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU Affero General Public License as published by
+  the Free Software Foundation, version 3 of the License.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  GNU Affero General Public License for more details.
+
+  You should have received a copy of the GNU Affero General Public License
+  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+-->
+<div class="container">
+  <div class="row" *ngFor="let field of entities | keyvalue; let i = index">
+    <div [class]="i%2 === 0 ? 'bg-light':''" class="font-weight-bold col-3 py-1" translate>
```

Comment:
`[class.bd-light]="!!(i%2)"` is prettier.
This class should be added on parent `div.row` element. All children div's will be colored